Xamarin.MacDev/MobileProvision.cs

```diff
@@ -57,7 +57,16 @@ static MobileProvision ()
 			if (Environment.OSVersion.Platform == PlatformID.MacOSX
 				|| Environment.OSVersion.Platform == PlatformID.Unix) {
 				string personal = Environment.GetFolderPath (Environment.SpecialFolder.UserProfile);
-				ProfileDirectory = Path.Combine (personal, "Library", "MobileDevice", "Provisioning Profiles");
+
+				// Xcode >= 16.x uses ~/Library/Developer/Xcode/UserData/Provisioning Profiles
+				string profileDir = Path.Combine (personal, "Library", "Developer", "Xcode", "UserData", "Provisioning Profiles");
+
+				if (!Directory.Exists (profileDir)) {
+					// Xcode < 16.x uses ~/Library/MobileDevice/Provisioning Profiles
+					profileDir = Path.Combine (personal, "Library", "MobileDevice", "Provisioning Profiles");
+				}
```

Comment:
Xcode 16's release notes says: "Xcode will still load previously downloaded profiles from the previous location.", this looks like it'll pick one or the other directory, not load from both.